.travis.yml

```diff
@@ -22,6 +22,19 @@ jobs:
       cache: pip
       install: pip install moban
       script: .ci/check_moban.sh
+    - stage: coala
+      <<: *disable_global
+      language: python
+      python: 3.6
+      cache: pip
+      install:
+        - cOALA_ZIP_URL=https://github.com/coala/coala/archive/master.zip
```

Comment:
You are using codeload for coala-bears, you might as well use the same for coala.

Reply:
moban needs to be synced on this repo, in that time you could test this out by cancelling the moban job (so that the coala job runs).